server/container_create.go

```diff
@@ -606,9 +606,17 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		if err != nil {
 			return nil, err
 		}
-		if err1 := os.MkdirAll(fp, 0644); err1 != nil {
-			return nil, err1
+		switch s.config.ImageVolumes {
+		case "mkdir":
+			if err1 := os.MkdirAll(fp, 0644); err1 != nil {
+				return nil, err1
+			}
+		case "ignore":
+			logrus.Debugf("Ignoring volume %v", dest)
+		default:
+			logrus.Warnf("Unknown image volumes setting: %v", s.config.ImageVolumes)
```

Comment:
This should probably be a failure, but this should be checked in the options parsings.